src/app/signal/signal-example1/signal-example1.component.scss

```diff
@@ -0,0 +1,9 @@
+.counterButton {
```

Comment:
style is not required, we can use the default material button with primary color